### ISSUE TITLE
[MsTeams][Sign in ] Open Sign in link in full window

### ISSUE THREAD

**Issue by gauravsaralMs** [microsoft/AdaptiveCards#4662]
**What platform is your feature request related to? (Delete other platforms).**

MsTeams

**Is your feature request related to a problem? Please describe.**
our service oauth page is large and needs more space to display than the default size of browser pop-up where sign in link is displayed.

**Describe the solution you'd like**
Open the sign-in link in a full windows browser instead of a smaller pop up

**Describe alternatives you've considered**
N/A

**Additional context**
Is it possible to make this pop-up full screen?

This is how we are sending the card to launch sign-in

``` json
{
  "type": "AdaptiveCard",
  "$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
  "version": "1.2",
  "body": [
    {
      "type": "TextBlock",
      "text": "Sign in card",
      "height": "stretch",
      "size": "ExtraLarge",
      "weight": "Bolder",
      "color": "Dark"
    }
  ],
  "actions": [
    {
      "type": "Action.Submit",
      "title": "Click me for signin",
      "data": {
        "msteams": {
          "type": "signin",
          "value": "Sing In URL"
        }
      }
    }
  ]
}
```

talking about the pop up here:
![image](https://user-images.githubusercontent.com/43203065/91277489-862b7500-e7a0-11ea-81e5-1688cdcbee2c.png)

**Comment by aosolis** [comment]
The short answer is no, they cannot be opened full screen. Here's how you can control the sizes:
- The size of signin popups can be controlled by adding width and height parameters to the query string of the URL to be opened in the popup. Teams will treat those as pixel sizes and open a popup of the corresponding size, up to a maximum of the 90% of the size of the main Teams window.
- Task module sizing is described here: https://docs.microsoft.com/en-us/microsoftteams/platform/task-modules-and-cards/what-are-task-modules#task-module-sizing. You can use the strings "small", "medium" and "large", or specify a pixel size. Large pixel sizes are capped at a percentage of the size of the main window (the task module is displayed *within* the window, so it cannot extend past it).

**Comment by gauravsaralMs** [comment]
@aosolis : 
for the sign-in card and for task module: what is the max value we can put for width and height parameters?

**Comment by aosolis** [comment]
You can put a really large value like 5000 pixels. We will cap it at 90% of the size of the main window.

**Comment by gauravsaralMs** [comment]
ok.. I actually tried with 25000 and it is not looking like 90%, there is growth for sure but not significant..
without change
![image](https://user-images.githubusercontent.com/43203065/91813107-c9c62900-ec4f-11ea-9023-91b46e51f3ea.png)
 with change
![image](https://user-images.githubusercontent.com/43203065/91813144-d5195480-ec4f-11ea-94b5-c78c50b4ac15.png)

**Comment by embailie** [comment]
@aosolis - it looks like the card is responsive to the changed configuration. Can we confirm this is the largest supported size for the task module?

**Comment by aosolis** [comment]
Yes, that's the max. I misspoke. The current max size for sign in popup is (Teams width - 400px) x (Teams height - 200px).
Note that this is not a documented contract, and the heuristic can change at any time,
